docs/get-started.md

```diff
@@ -1,42 +1,121 @@
-## Get Started
+# Get Started with Open Service Mesh Arc extension
 Use the following scripts to publish chart, enable connected cluster, and add OSM Arc extension.
 
 ## Prerequisites
 - Running Kubernetes cluster
-- Azure CLI
+- [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
-### Set env vars
-The OSM Azure repo relies on environment variables to make it usable on your localhost. The root of the repository contains a file named `.env.example`. Copy the contents of this file into `.env`
+## Add Azure CLI extensions
+
+> Note: Get latest wheels: https://github.com/Azure/azure-arc-kubernetes-preview/tree/master/extensions
+
+1. Install the latest [connectedk8s](https://docs.microsoft.com/en-us/cli/azure/connectedk8s?view=azure-cli-latest).
 ```bash
-cat .env.example > .env
+WHEEL="<location of the Azure CLI Python wheel file>"
+az extension add --source $WHEEL --yes || true
 ```
-The various environment variables are documented in the `.env.example` file itself. Modify the variables in `.env` to suite your environment.
 
-Note: Ensure the region specified in the .env file has the latest OSM arc extension available. Supported regions listed [here](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/connect-cluster#supported-regions).
+2. Install the latest [k8s-extension](https://docs.microsoft.com/en-us/cli/azure/k8s-extension?view=azure-cli-latest).
+```bash
+WHEEL="<location of the Azure CLI Python wheel file>"
+az extension add --source $WHEEL --yes || true
+```
+
+## Enable Arc Features
+The following steps need to be run only once for a given subscription:
+
+_Optional_: set default Azure subscription for the `az` CLI:
+```bash
+az account set --subscription=<your-Azure-subscription-ID>
+```
+
+```bash
+az feature register \
+  --namespace Microsoft.Kubernetes \
+  --name previewAccess
+```
+
+```bash
+az feature register \
+  --namespace Microsoft.KubernetesConfiguration \
+  --name sourceControlConfiguration
+```
+
+```bash
+az feature register \
+  --namespace Microsoft.KubernetesConfiguration \
+  --name extensions
```

Comment:
This is not needed any more as exetension is on by default now. Above list covers all the updated feature flags

Reply:
Removed!